packages/ember-runtime/lib/mixins/observable.js

```diff
@@ -306,6 +306,30 @@ export default Mixin.create({
     order and cause notifications to be delivered more often than you would
     like.
 
+    Property change notifications do not propogate over bindings if the object
+    returned is the same object (compared with ===). When a binding is created,
+    an observer is added on both the "from" and "to" paths. When
+    propertyDidChange is called on the "from" side, it triggers the "from"
+    observer which schedules a synchronization of the binding in the run loop
+    (into the sync queue). Synchronization just means to call set on the "to"
+    side of the binding with the value of the "from" side. This set call becomes
+    a no op if the value it is being set to is the same as it was previously.
```

Comment:
Although the above is all true, I don't see how if I didn't know how this process already worked, I'd understand the process from this text. It's probably too "inside baseball". What is the goal you're trying to achieve here?